backend/app/services/knowledge/knowledge_service.py

```
@@ -1692,37 +1707,37 @@ def get_all_knowledge_bases_grouped(
             .all()
         )
 
-        # Get organization namespace info
-        org_namespace = (
+        organization_namespaces = (
             db.query(Namespace)
             .filter(
                 Namespace.level == GroupLevel.organization.value,
                 Namespace.is_active == True,
```

Ruff (0.15.7)

[error] Avoid equality comparisons to `True`; use `Namespace.is_active:` for truth checks

Replace with `Namespace.is_active`

(E712)